Makefile

```diff
@@ -227,6 +227,8 @@ build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata
 build-cluster-templates: $(RELEASE_DIR) cluster-templates
 	cp $(BYOH_TEMPLATES)/v1beta1/templates/docker/cluster-template.yaml $(RELEASE_DIR)/cluster-template-docker.yaml
 	cp $(BYOH_TEMPLATES)/v1beta1/templates/vm/cluster-template.yaml $(RELEASE_DIR)/cluster-template.yaml
+	cp $(BYOH_TEMPLATES)/v1beta1/templates/vm/cluster-template-topology.yaml $(RELEASE_DIR)/cluster-template-topology.yaml
+	cp $(BYOH_TEMPLATES)/v1beta1/templates/vm/clusterclass-quickstart.yaml $(RELEASE_DIR)/clusterclass-quickstart.yaml
```

Comment:
And we should be generating these as well, right?
Generating cluster templates happens in this `make` task
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/f8aa753e622709b5afc7156c0c33474eb51fee9e/Makefile#L168

Reply:
These files are present in `$(BYOH_TEMPLATES)/v1beta1/templates/vm/` in there final form so generation with kustomize should not be needed.

Reply:
Hmmm I don't know if we should be relying on that? How do other providers do this?
If you feel, they are good enough, we should follow the same for other artifacts too, currently those are being generated on the fly.

I don't have any strong opinions, but wondering if anybody fails to update cluster-template in the repo, that will end up in the release artifacts.

Reply:
The templates in path `$(BYOH_TEMPLATES)/v1beta1/templates/docker` and  `$(BYOH_TEMPLATES)/v1beta1/templates/vm` are part of release artifacts, these are kept up-to-date and used during release. 
We do similar thing in CAPA, the [templates](https://github.com/kubernetes-sigs/cluster-api-provider-aws/tree/main/templates) folder contains the release artifacts, these are updated manually and used during release.
The only difference is that we generate our templates using `kustomize build` while CAPA does not use kustomize in their templates folder, but this should be fine.

Reply:
Alright, sounds good 👍🏽